src/types/libdef/npm-custom/react-testing-library_v5.x.x.js

```diff
@@ -1,5 +1,6 @@
 // flow-typed signature: 187ae8536dde5ccffcfaa74225e94ed0
```

Comment:
I missed some functions in the libdef so I had to change it. I plan to upstream these changes with others but didn't want to depend on this for the PR.